completed/modeler.js

```diff
@@ -1,5 +1,5 @@
 import axios from "axios";
-import { getAccessToken } from "../auth.js";
+import { getAccessToken } from "./auth.js";
```

Comment:
@christinaausley I modified all references to `auth.js` from the perspective of the root directory, where the user would be building their API. From the perspective of this `completed/` directory, this file wouldn't find `./auth.js`; but if they were to compare their code in the root folder to this file, the path would match, and they could also copy/paste these contents into their working file and it would work. 

That was kind of hard to explain and I don't think I did a great job; if this is still confusing, or you disagree, let me know.